packages/core/integration-tests/test/css.js

```diff
@@ -1,91 +1,64 @@
-  it('should produce two bundles when importing a CSS file', async function() {
+const {
+  bundle,
+  run,
+  assertBundles,
+  assertBundleTree,
+  rimraf,
+  ncp
+} = require('./utils');
+
+describe('css in v2', () => {
+  it('should produce two bundles when importing a CSS file', async () => {
     let b = await bundle(path.join(__dirname, '/integration/css/index.js'));
 
-    await assertBundleTree(b, {
-      name: 'index.js',
-      assets: ['index.js', 'index.css', 'local.js', 'local.css'],
-      childBundles: [
-        {
-          name: 'index.map'
-        },
-        {
-          name: 'index.css',
-          assets: ['index.css', 'local.css'],
-          childBundles: [
-            {
-              type: 'map'
-            }
-          ]
-        }
-      ]
-    });
+    await assertBundles(b, [
+      {
+        name: 'index.js',
+        assets: ['index.js', 'local.js']
+      },
+      {
+        name: 'index.css',
+        assets: ['index.css', 'local.css']
+      }
+    ]);
 
     let output = await run(b);
     assert.equal(typeof output, 'function');
     assert.equal(output(), 3);
   });
 
-  it('should support loading a CSS bundle along side dynamic imports', async function() {
+  it('should support loading a CSS bundle along side dynamic imports', async () => {
     let b = await bundle(
       path.join(__dirname, '/integration/dynamic-css/index.js')
     );
 
-    await assertBundleTree(b, {
-      name: 'index.js',
-      assets: [
-        'index.js',
-        'index.css',
-        'bundle-loader.js',
-        'bundle-url.js',
-        'js-loader.js',
-        'css-loader.js'
-      ],
-      childBundles: [
-        {
-          type: 'css',
-          name: 'index.css',
-          assets: ['index.css'],
-          childBundles: [
-            {
-              type: 'map'
-            }
-          ]
-        },
-        {
-          name: 'index.js.map'
-        },
-        {
-          type: 'js',
-          assets: ['local.css', 'local.js'],
-          childBundles: [
-            {
-              type: 'css',
-              assets: ['local.css'],
-              childBundles: [
-                {
-                  type: 'map'
-                }
-              ]
-            },
-            {
-              type: 'map'
-            }
-          ]
-        }
-      ]
-    });
+    await assertBundles(b, [
+      {
+        name: 'index.js',
+        assets: [
+          'bundle-loader.js',
+          'bundle-url.js',
+          'css-loader.js',
+          'index.js',
+          'js-loader.js',
+          'JSRuntime.js'
+        ]
+      },
+      {name: /local\.[0-9a-f]{8}\.js/, assets: ['local.js']},
+      {name: /local\.[0-9a-f]{8}\.css/, assets: ['local.css']},
+      {name: 'index.css', assets: ['index.css']}
+    ]);
 
     let output = await run(b);
     assert.equal(typeof output, 'function');
     assert.equal(await output(), 3);
   });
+});
 
+describe.skip('css', function() {
```

Comment:
should the old tests be removed/moved?

Reply:
I'll be getting more of them passing soon in following PRs 😃